### PR TITLE
fix textlint warnings

### PR DESCRIPTION
**Pull request by shogo82148**
some URLs return 301 responses, so I replaced them with their corresponding redirect destinations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -830,8 +830,8 @@ environment variables.
 | ---- | ----------- |
 | `CI_PULL_REQUEST` | Pull Request number (e.g. 14) |
 | `CI_COMMIT`       | SHA1 for the current build |
-| `CI_REPO_OWNER`   | repository owner (e.g. "haya14busa" for https://github.com/haya14busa/reviewdog) |
-| `CI_REPO_NAME`    | repository name (e.g. "reviewdog" for https://github.com/haya14busa/reviewdog) |
+| `CI_REPO_OWNER`   | repository owner (e.g. "haya14busa" for https://github.com/reviewdog/reviewdog) |
+| `CI_REPO_NAME`    | repository name (e.g. "reviewdog" for https://github.com/reviewdog/reviewdog) |
```
Comment (shogo82148) [inline README.md:834]: `haya14busa` is old repository owner, it is `reviewdog` now.

Comment (shogo82148) [reply README.md:834]:
```suggestion
| `CI_REPO_OWNER`   | repository owner (e.g. "reviewdog" for https://github.com/reviewdog/errorformat) |
| `CI_REPO_NAME`    | repository name (e.g. "errorformat" for https://github.com/reviewdog/errorformat) |
```

**Review by i-sevostyanov — APPROVED** [review]
Nice!

**Comment by shogo82148** [comment]
thanks!

**Pull request merged**